### PR TITLE
incorporate new register qaqc service and register base candidate

### PR DESCRIPTION
**Pull request by dschristianson**
## PR Self Evaluation

- [x] I have reviewed my code to check that it contains no sensitive information.
~- [ ] I have commented my code, particularly in hard-to-understand areas~
- [x] My changes generate no new warnings
- [x] I have added tests or modified existing tests that prove my fix is effective or that my feature works
- [x] Existing unit tests pass locally with my changes

### CODE REVIEW

File: processing/report_status.py [hunk 1]
```diff
@@ -98,13 +100,35 @@ def enter_new_state(self, process_id, state_id):
...
+            prior_process_id=None, zip_process_id=None):
+        register_format_qaqc_ws = f'{self.register_qaqc_ws}/format_qaqc'
+        req_data = {'site_id': site_id,
+                    'upload_id': upload_id,
+                    'process_timestamp': process_timestamp,
+                    'code_version': self.code_version,
+                    'processor': getpass.getuser()}
+
+        if prior_process_id:
+            req_data.update({'prior_process_id': prior_process_id})
```
Comment (yocheah) [inline processing/report_status.py:114]: You should be able to just do update(prior_process_id=prior_process_id)

File: processing/report_status.py [hunk 2]
```diff
@@ -123,7 +147,7 @@ def register_base_files(self, proc_id, file, input_info):
                           'start': part.start,
                           'end': part.end})
         req_data['upload_files'] = parts
-        url = '{u}/{i}'.format(u=self.register_base_ws, i=proc_id)
+        url = '{u}/{i}'.format(u=self.register_base_candidate, i=proc_id)
```
Comment (yocheah) [inline processing/report_status.py:150]: Can we not use f-strings here?

File: processing/report_status.py (hunk 1 above)
Comment (dschristianson) [reply processing/report_status.py:114]: done

File: processing/report_status.py (hunk 2 above)
Comment (dschristianson) [reply processing/report_status.py:150]: all is changed properly now.

**Review by yocheah — APPROVED** [review]

**Pull request merged**